kubemarine/resources/configurations/compatibility/kubernetes_versions.yaml

```diff
@@ -1,9 +1,5 @@
 # This section can be changed manually, but it is also synchronized automatically with the 'compatibility_map' section below.
 kubernetes_versions:
-  v1.21:
-    supported: true
-  v1.22:
-    supported: true
```

Comment:
Since we stop support for these version, we can get rid of very many obsolete code in `kubemarine/kubernetes/__init__.py`